hydro_test_local/src/local/chat_app.rs

```diff
@@ -51,7 +51,7 @@ pub fn chat_app<'a>(
     flow.compile_no_network::<SingleProcessGraph>()
 }
 
-#[stageleft::runtime]
+#[cfg(stageleft_runtime)]
```

Comment:
Is `#[stageleft::runtime]` changed to `#[cfg(stageleft_runtime)]` now, in places where it is still needed?

Reply:
Yep, reason for this is then we can put it on `mod` declarations.